suites/nautilus/dashboard/sanity_containerized_ceph_dashboard.yaml

```diff
@@ -0,0 +1,104 @@
+tests:
+   - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+
+   - test:
+        name: containerized ceph with dashboard
+        polarion-id: CEPH-83573534
+        module: test_ansible.py
+        config:
+          is_mixed_lvm_configs: True
+          ansi_config:
+              ceph_test: True
+              ceph_origin: distro
+              ceph_stable_release: nautilus
+              ceph_repository: rhcs
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              journal_size: 1024
+              ceph_stable: True
+              ceph_stable_rh_storage: True
+              containerized_deployment: true
+              copy_admin_key: true
+              ceph_docker_registry: "registry.redhat.io"
+              ceph_docker_registry_auth: true
+              ceph_docker_registry_username: <provide_username>
+              ceph_docker_registry_password: <provide_password>
+              dashboard_enabled: True
+              dashboard_admin_user: <provide_username>
+              dashboard_admin_password: <provide_password>
+              grafana_admin_user: <provide_username>
+              grafana_admin_password: <provide_password>
+              node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.1
+              grafana_container_image: registry.redhat.io/rhceph/rhceph-4-dashboard-rhel8:4
+              prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:4.1
+              alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:4.1
```

Comment:
ceph_origin: distro,  unreleased rhceph version

with 
grafana_container_image: registry.redhat.io/rhceph/rhceph-4-dashboard-rhel8:4
released dashboard version

is it intented @sunilangadi2 ?

Reply:
@XDsubhash modified ceph origin to **repository** with the same released grafana image

Reply:
@sunilangadi2 is this suite supposed to test GAed versions? or internal builds?

Reply:
@XDsubhash this dashboard suite requires `registry registry.redhat.io` and suppose if i gave this registry  `registry-proxy.engineering.redhat.com` then dashboard installation fails so currently this suite works fine with GAed version once we figure out how to do automation of podman login of this registry registry.redhat.io to all nodes https://github.com/red-hat-storage/cephci/issues/189 then this suite also works for distro (means internal builds)

Reply:
@sunilangadi2 #189 issue is to handle the same registry(registry.redhat.io) that  you are addressing in previous comment.

Reply:
@sunilkumarn417 yes if we resolve this issue https://github.com/red-hat-storage/cephci/issues/189 then we can pass this registry `registry-proxy.engineering.redhat.com` in cli

Reply:
issue linked.